src/main/perl/Lock.pm

```diff
@@ -193,6 +216,54 @@ sub _try_lock
     return SUCCESS;
 }
 
+
+# Before CAF PR#132, CAF::Lock used the presence of the lock file as
+# condition to determine if the lock was taken or not.
+# The file would hold the PID of the process that had the lock
+# (and unlock removed the file).
+# FORCE_IF_STALE would force
```

Comment:
I have the feeling that part of the sentence is missing... or that this is a leftover...